requirements-dev.txt

```
@@ -16,13 +16,11 @@ scikit-learn>=1.3,<2
 xgboost>=0.90,<2
 lightgbm>=2,<4
 
-# PyTorch doesn't support Python 3.11 yet (pytorch/pytorch#86566)
-
 # Elasticsearch uses v1.13.1 of PyTorch
-torch>=1.13.1,<2.0; python_version<'3.11'
+torch>=1.13.1,<2.0
```

I think it is worth keeping the restriction for the unreleased Python '3.12' as we do not know if that will be compatible with PyTorch yet. 

```suggestion
torch>=1.13.1,<2.0; python_version<'3.12'
```

---

> I think it is worth keeping the restriction for the unreleased Python '3.12' as we do not know if that will be compatible with PyTorch yet.

Good point, I applied the suggestion.